src/traits/msg_sign.rs

```diff
@@ -36,6 +37,7 @@ impl_msg_sign!(GatewayRegionParamsUpdateReqV1, signature);
 impl_msg_sign!(BlockchainStateChannelPacketV1, signature);
 impl_msg_sign!(BlockchainStateChannelOfferV1, signature);
 impl_msg_sign!(BlockchainTxnStateChannelCloseV1, signature);
+impl_msg_sign!(PacketRouterPacketUpV1, signature);
```

Comment:
can we rename the proto to `PacketRouterUplinkV1` ?

Reply:
I do like that name. @macpie thoughts?

Reply:
I like the `v1` but not `uplink` because not only `uplinks` come threw there but `joins` as well